CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.3] - 2024/09/20
```

Comment:
Thanks for adding 😄. Could you add the link to this version/tag as well in this file.

Reply:
Of course: [07a8762](https://github.com/phac-nml/gasnomenclature/pull/27/commits/07a876287ae5c4aaf1d1e3f146306ce9ed08cf6a)